src/ebfm/LOOP_write_to_file.py

```diff
@@ -7,7 +7,7 @@
 from netCDF4 import Dataset, date2num
 
 
-def main(OUTFILE, io, OUT, grid, t, time, C):
+def main(OUTFILE, io, OUT, grid, t, time, C, gridtype):
```

Comment:
Is it possible to use `grid["input_type"]` here? Introducing a new type is no problem in my opinion. Extending the interface is something I would only do if necessary.